beaker-vue/eslint.config.ts

```diff
@@ -16,7 +16,7 @@ export default defineConfigWithVueTs(
     files: ['**/*.{ts,mts,tsx,vue}'],
   },
 
-  globalIgnores(['**/dist/**', '**/dist-ssr/**', '**/coverage/**']),
+  globalIgnores(['**/dist/**', '**/dist-ssr/**', '**/coverage/**', 'extractRoutes.ts']),
```

Comment:
this is basically a drive-by changes, since eslint for UI was trying to lint extractRoutes.ts and failing